src/ui/components/atoms/logo/style.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../../../globals/variables';
+@import '../../../../globals/mixins';
+
+.logo-image {
+  border-radius: 0 0 1.5rem;
+  height: 51px;
+  padding-top: 0;
+  width: 130px;
```

Comment:
This measure is bab the correct is 130 - 27 = 103 in mobile and ins desktop is other